<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>room_1</class>
 <widget class="QWidget" name="room_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="room1_picture">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>11</y>
     <width>781</width>
     <height>581</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/picture_room1.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_jeu1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Trie tes déchets !</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_jeu2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Le sais-tu ?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_jeu3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Décode !</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ans1_pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>L'Italie</string>
   </property>
  </widget>
  <widget class="QLabel" name="question_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>701</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>Il existe dans l’océan Pacifique une immense nappe de déchets, elle fait la taille de :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ans2_pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>490</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>La France</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ans3_pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>490</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>L'australie</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultmessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>421</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/picture_bravo.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="next_pushButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>530</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Des scientifiques ont découvert au nord de l’océan Atlantique une &lt;span style=&quot; font-weight:700;&quot;&gt;immense nappe de déchets&lt;/span&gt;, flottant sur l’océan : des capsules, des sacs plastique, des bouchons et des ordures en tout genre s’étendent sur près de &lt;span style=&quot; font-weight:700;&quot;&gt;700 000 kilomètres carrés&lt;/span&gt;, soit la taille de la France. Cette poubelle géante représente un danger pour toutes les espèces marines.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="nextRoom_pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Youpi !
 Salle suivante, allons sauver le monde !</string>
   </property>
  </widget>
  <widget class="QLabel" name="completed_picture">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/picture_room_end.JPG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
